Post/migrations/0002_test_data.py

```diff
@@ -1,35 +1,42 @@
 from django.db import migrations, transaction
+from django.contrib.auth.models import User
 
 
 class Migration(migrations.Migration):
     dependencies = [
+        ('users', '0001_initial'),
+        ('users', '__latest__'),
         ('Post', '0001_initial'),
     ]
 
     def generate_data(apps, schema_editor):
```

Comment:
please refer commenting test_data. i would advise implement it the same.
user_list, posts_list and for loop under with transaction

Reply:
I changed the file as you asked.